.github/CODEOWNERS

```diff
@@ -58,8 +58,8 @@
 /tensorflow_addons/image/tests/distance_transform_test.py @mels630
 /tensorflow_addons/image/distort_image_ops.py @windqaq
 /tensorflow_addons/image/tests/distort_image_ops_test.py @windqaq
-/tensorflow_addons/image/filters.py @mainak431
-/tensorflow_addons/image/tests/filters_test.py @mainak431
+/tensorflow_addons/image/filters.py @mainak431 @ghosalsattam
+/tensorflow_addons/image/tests/filters_test.py @mainak431 @ghosalsattam
```

Comment:
These changes should be reverted. The `CODEOWNERS` file contains people from the team at Google that know everything about the code in the corresponding entries.

Contributors can contribute to code without being listed in `OWNERS` files

Reply:
@mihaimaruseac I have removed it.

Reply:
@mihaimaruseac For TensorFlow Addons, maintainers are community members (typically not Googlers). For new contributions we require contributors be willing to maintain the code and add themselves to the CODEOWNERS. Perhaps mistaken which repository this PR was for?

@ghosalsattam Please revert your revert  :laughing:

Reply:
@seanpmorgan I have added the codeowners again.